backend/src/controllers/activitiesController.ts

```diff
@@ -21,7 +21,8 @@ export const getActivitiesByUserId = async(req: Request, res: Response) => {
             where: {user_id: Number(userId)},
             include: {route: true},
         });
-
+        console.log('activities:', activities);
+        
```

Comment:
@Higesennine I can see this log has appeared only once when logging in to this website
<img width="1440" alt="Screenshot 2024-12-03 at 19 38 01" src="https://github.com/user-attachments/assets/cdd7ce36-25a3-49b3-bfbd-39207206ce90">

Reply:
I think one of this problem is from .env file in frontend.
I sent it in Slack, so could you try again after check slack?